lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
The `sort -u` will certainly get removed/skipped with #1902, so it is (maybe already?) specific to bash.

Would be nice to have your example as a test, in case you have not added it already.